yaksh/test_models.py

```diff
@@ -2130,3 +2131,22 @@ def test_course_is_complete(self):
 
         # Test get course grade after completion
         self.assertEqual(self.course.get_grade(self.answerpaper1.user), 'B')
+
+
+class FileUploadTestCases(unittest.TestCase):
+    def setUp(self):
+        self.question = Question.objects.get(summary='Q1')
+        self.filename = "uploadtest.txt"
+        self.uploaded_file = SimpleUploadedFile(self.filename, b'Test File')
+        self.file_upload = FileUpload.objects.create(
+            file=self.uploaded_file,
+            question=self.question
+        )
+
+    def test_get_file_name(self):
+        print((self.file_upload.file.path))
+        self.assertEqual(self.file_upload.get_filename(), self.filename)
+
+    def tearDown(self):
+        os.remove(self.file_upload.file.path)
```

Comment:
check if the file exists before removing. It might happen sometimes that the file may not be present. Just a safety check.

Reply:
Fixed in subsequent PR #575